src/CMakeLists.txt

```diff
@@ -59,6 +59,15 @@ set_target_properties(morphio_obj
   CXX_EXTENSIONS NO
   )
 
+target_compile_options(morphio_obj
```

Comment:
There is already a way to activate Werror (feel free to add additionnal flags there):
https://github.com/BlueBrain/MorphIO/blob/master/.ContinousIntegration.cmake

But I don't think it should be a default for people using the soft (for example through pip install) as opposed to developpers.

Reply:
I'm inclined to disagree here, since this can catch errors when locally making changes (before the CI is even involved). I'd think that in general, you would want to have pip install a pre-built binary anyways?